pa4/checker-pa4.ml

```diff
@@ -251,6 +262,12 @@ let get_own_attributes input_class = begin
 			match feature with
 				| Attribute ((attribute_loc, attribute_name), _, _) -> 
 					(* Check for duplicate method in the same class. *)
+				if attribute_name = "self" then begin
```

Comment:
This whole block should be indented one more so that it's clear that it belongs side of this case of the match expression.